storage/src/main/resources/db/migration/V20190927_894__Add_deploy_dependencies.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE deploy_headers
+(
+    hash                 BLOB PRIMARY KEY NOT NULL,
+    account              BLOB             NOT NULL,
+    timestamp_millis     INTEGER          NOT NULL,
+    header               BLOB             NOT NULL,
+    FOREIGN KEY (hash) REFERENCES deploys (hash)
+) WITHOUT ROWID;
```

Comment:
```suggestion
);
```
I think in this case it would be best not to use this feature. According to point section 4 at https://www.sqlite.org/withoutrowid.html the rows should be much smaller than what storing a full deploy header requires.